demo.py

```diff
@@ -0,0 +1,107 @@
+import gradio as gr
+import requests
+import base64
+import tempfile
+import json
+import os
+
+
+SERVER_URL = 'http://localhost:8000'
+OUTPUT = "./demo_outputs"
+cloned_speakers = {}
+
+print("Preparing file structure...")
+if not os.path.exists(OUTPUT):
+    os.mkdir(OUTPUT)
```

Comment:
Side effects should never run top-level, always be contained inside an `if __name__ == '__main__':` block.

Reply:
Yeah but I couldn't make it work, here I define STUDIO_SPEAKERS  for example that I use below, if I move it in the if main I get `NameError: name 'STUDIO_SPEAKERS' is not defined`. Maybe we can update in the next PR?